presto-hive/src/main/java/com/facebook/presto/hive/BackgroundHiveSplitLoader.java

```diff
@@ -645,4 +661,25 @@ private static String getPartitionLocation(Table table, Optional<Partition> part
         }
         return partition.get().getStorage().getLocation();
     }
+
+    private static Optional<Domain> getPathDomain(TupleDomain<HiveColumnHandle> effectivePredicate)
+    {
+        if (!effectivePredicate.getDomains().isPresent()) {
+            return Optional.empty();
+        }
+
+        return effectivePredicate.getDomains().get().entrySet().stream()
+                .filter(entrySet -> isPathColumnHandle(entrySet.getKey()))
+                .findFirst()
+                .map(entrySet -> entrySet.getValue());
```

Comment:
It's an `entry`, not `entrySet`. In addition, you can write `Map.Entry::getValue` here.